stack.Makefile

```diff
@@ -50,7 +50,7 @@ bundle: $(STACK_PACKAGE_REGISTRY)
 	# be to cat all of the files into a single crd.yaml.
 	find $(CRD_DIR) -type f -name '*.yaml' | \
 		while read filename ; do cat $$filename > \
-		$(STACK_PACKAGE_REGISTRY)/resources/$$( basename $${filename/.yaml/.crd.yaml} ) \
+		$(STACK_PACKAGE_REGISTRY)/resources/$$( basename $$(echo $$filename | sed s/.yaml$$/.crd.yaml/)) \
```

Comment:
any ideas why you didn't hit this while building the cloud provider stack repos?  e.g. stack-gcp uses the same command:
https://github.com/crossplaneio/stack-gcp/blob/master/Makefile#L137